core/src/main.cpp

```diff
@@ -847,6 +846,36 @@ auto load_oidc_configuration(const json& _config, json& _oi_config, json& _endpo
 	return true;
 } // load_oidc_configuration
 
+auto load_user_mapping_plugin(const json& _user_mapping_config) -> bool
```

Comment:
I doubt that this offers much, but would it be more interesting to return an `int` or something from this function so the caller can know the error which occurred, if any? What you have is fine, just throwing it out there.

Reply:
While we can return the status/error code from this function, we cannot act on any of the codes, aside from the fact that the code is zero/non-zero. This behavior is also seen within `load_user_mapping_plugin(...)` itself:

https://github.com/irods/irods_client_http_api/blob/f5f4bf3fabba9ba295e44b8c2cd63510115ac828/core/src/main.cpp#L870-L876

This is because we don't guarantee that these plugins, or any plugins developed by others, follow any sort of error code convention apart from a non-zero code meaning something bad happened.

Reply:
Okay, makes sense to me. Resolving.